README.md

```diff
@@ -19,19 +20,35 @@ The server exposes APIs that are consumed by the web app and the mobile app.
    ```
    SECRET = secure_server_secret
    MONGODB_URI = mongodb://localhost:27017/boldo
-
+   
    # ###################### Online ######################
    # CLIENT_ADDRESS = https://boldo.penguin.software
    # KEYCLOAK_ADDRESS = https://sso-test.pti.org.py/auth
    IHUB_ADDRESS = https://sso-test.pti.org.py/api
-
    # ###################### Local ######################
    CLIENT_ADDRESS = http://localhost:3000
    KEYCLOAK_ADDRESS = http://localhost:8080/auth
+   BOLDO_CORS = http://localhost:3000
    ```
 
 4. `npm run dev` - to start server on [localhost:8008](http://localhost:8008)
 
+## Docker Initialization
```

Comment:
This not only described `Initialization` but also how to ru it. Use a more precise headline.
Suggestion: `Run with Docker`